index.bs

```diff
@@ -1278,6 +1281,38 @@ Gets the {{Error}} object passed to {{SensorErrorEventInit}}.
 </div>
 
 
+<h3 dfn>Security Check</h3>
+
+<div algorithm="security check">
+
+    : input
+    :: None
+    : output
+    :: A string whose value is either "secure" or "unsecure".
+
+    1.  Let |document| be the [=top-level browsing context=]'s [=active document=].
+    1.  Let |current_visibility_state| be the result of running
+        the [= steps to determine the visibility state=] of |document|.
+    1.  If |current_visibility_state| is not "visible",
+        then return "unsecure".
+    1.  If the [=currently focused area=] of the current [=top-level browsing context=]
+        is a [=nested browsing context=] whose [=active document=]'s [=origin=]
+        is not [=same origin-domain=] as |document|'s [=origin=],
+        then return "unsecure".
+    1.  If the [=currently focused area=] is in a different [=top-level browsing context=]
```

Comment:
Yeah let's simplify this as there might be additional security concerns here (such as running sensors in an unfocused window of the same origin to avoid triggering visual clues in the browser chrome on the focused context).